build/run-ci.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 # run STF CI setup in CRC (already provisioned)
 - hosts: localhost
-  gather_facts: no
+  gather_facts: yes
```

Comment:
Is this required or was this left as part of testing?

Reply:
It's needed because we use ansible_env for a default somewhere.

It will be removed again soon, hopefully